docs/en/maintain/backup.md

```diff
@@ -0,0 +1,202 @@
+> showopstatus
+  op_id  op_type              name  pid  status  start_time      execute_time  end_time        cur_task
+---------------------------------------------------------------------------------------------------------
+  54     kUpdateTableAliveOP  flow  0    kDone   20180824195838  2s            20180824195840  -
+  55     kChangeLeaderOP      flow  0    kDone   20180824200135  0s            20180824200135  -
+  56     kOfflineReplicaOP    flow  1    kDone   20180824200135  1s            20180824200136  -
+  57     kUpdateTableAliveOP  flow  0    kDone   20180824200212  0s            20180824200212  -
+  58     kRecoverTableOP      flow  0    kDone   20180824200623  1s            20180824200624  -
+  59     kRecoverTableOP      flow  1    kDone   20180824200623  1s            20180824200624  -
+  60     kReAddReplicaOP      flow  0    kDone   20180824200624  9s            20180824200633  -
+  61     kReAddReplicaOP      flow  1    kDone   20180824200624  9s            20180824200633  -
+> showtable
+>showtable
+  name    tid  pid  endpoint            role      ttl       is_alive  compress_type  offset   record_cnt  memused
+----------------------------------------------------------------------------------------------------------------------
+  flow    4   0    172.27.128.32:8541  follower  0min       yes       kNoCompress    0        0           0.000
+  flow    4   0    172.27.128.33:8541  follower  0min       yes       kNoCompress    0        0           0.000
+  flow    4   0    172.27.128.31:8541  leader    0min       yes       kNoCompress    0        0           0.000
+  flow    4   1    172.27.128.33:8541  leader    0min       yes       kNoCompress    0        0           0.000
+  flow    4   1    172.27.128.31:8541  follower  0min       yes       kNoCompress    0        0           0.000
+  flow    4   1    172.27.128.32:8541  follower  0min       yes       kNoCompress    0        0           0.000
+```
+
+If showtable turns to yes, it means that the recovery has been successful. If some shards fail to recover, you can execute recovertable separately. The command format is: recovertable table_name pid endpoint
+
+**Note: offlineendpoint must be executed once before recoverendpoint is executed**
+
+### One-click recovery
+
+If the cluster is restarted after being offline at the same time, autofailover cannot automatically restore data, and a one-click recovery script needs to be executed.
+
+#### Steps:
+
+1. Modify env.sh in the tools directory of the openmldb package. There are 3 parameters in the configuration file
+
+* openmldb_bin_path specifies the openmldb bin path
+* zk_cluster specifies the address of the zk cluster
+* zk_root_path specifies the address of the zk root path
+
+2. Execute ./recoverdata.sh
+3. Log in to ns_client, and showopstatus to check the execution progress of related ops.
+
+### Manual data recovery
+
+Applicable scenario: The node where all replicas of a table shard are located is down
```

Comment:
The node where all replicas of a table shard are located is down
=>
The nodes where all replicas of a partition are located are down.